lib/LaTeXML/Package/TeX.pool.ltxml

```diff
@@ -907,19 +908,7 @@ DefMacro('\meaning Token', sub {
         $meaning_cc .= ' ' if $meaning_cc;    # append space separator if defined
         $meaning = $meaning_cc . $char; }
       elsif ($type =~ /register$/i) {
-        my $value         = $definition->valueOf;
-        my $register_type = lc(ref $value);
-        my $prefix        = '\count';
-        if ($register_type) {
-          if ($register_type =~ /glue/) {
-            $prefix = '\skip'; }
-          elsif ($register_type =~ /dimension/) {
-            $prefix = '\dimen'; }
-          elsif ($register_type =~ /tokens/) {
-            $prefix = '\toks'; } }
-        my $literal_value = $register_type && $value->valueOf;
-        # Should we be more careful to distinguish between latex and tex counters?
-        $meaning = $prefix . $literal_value; }
+        $meaning = $$definition{address} || $definition->getCSName; }
```

Comment:
Could one add a new `Register::getCSName` which returns the `{address}` field? That may make porting this a little easier in well-typed languages as just `$meaning = $definition->getCSName` on any `Definition`.

Reply:
It probably needs an accessor (but not getCSName), even though this probably should be the only place that should know that it has an address stored internally.

Reply:
Or even possibly a `getMeaning` (non-mathematical), and shift the burden do the Definitions?

Reply:
ok, done